frontend/e2e/src/support/pages/common.constants.ts

```diff
@@ -44,7 +44,7 @@ export const Resource = {
     } else {
       cy.get(Resource.buttons.details).first().click();
     }
-    cy.url().should('match', /\/\d+$/);
+    cy.url().should('match', /\/.+$/);
```

Comment:
Wszystko git, tylko co do tego mam wątpliwości, bo teraz jakby redirect nie zadziałał z jakiegoś powodu to np. `/tickets` też pasuje w tym regexie. Gdy były tylko liczby to \d+ sprawdzał id obiektu. Pewnie prędzej czy później test tak czy siak by sie wywalił jeśli przejście na szczególy nie zadziała

Lepiej byłoby pewnie coś takiego:
`/\/(\d+|.+\-.+)$/`